doc/classes/ItemList.xml

```diff
@@ -406,19 +406,28 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="empty_clicked">
+			<argument index="0" name="at_position" type="Vector2" />
+			<argument index="1" name="mouse_button_index" type="int" />
+			<description>
+				Triggered when a left or right mouse click is issued within the rect of the list but on empty space.
```

Comment:
Would it make sense to also allow middle mouse button? Nothing needs it in the engine right now but now that this has been made more generic, it could be handled too for people who want to implement specific functionality for each mouse button.

Same goes for `item_clicked`.